mssql/internal/sqlcmd/Database.go

```diff
@@ -0,0 +1,71 @@
+package sqlcmd
+
+type Database struct {
+	Id                 int
+	Name               string
+	Ownersid           string
+	CompatibilityLevel string
+	Collation_name     string
+	IsReadOnly         bool
+	RecoveryModel      string
+	FullTextEnabled    string
+}
+
+func (c *SqlCommand) GetDatabase(id string, where string) (*Database, error) {
```

Comment:
general comment;
maybe rename this local var 'c' to something more declarative.
that will help other contributors to read & understand the code a bit better
maybe 'sqlClient' or something

Reply:
Will do this in a separate pr because it is also used outside database